client/index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width-device-width, initial-scale=1.0" />
+        <meta http-equiv="X-UA-Compatbile" content="ie=edge" />
+        <link rel="stylesheet" href="style.css" />
+        <script defer src="homepage.js"></script>
+        <title>Google</title>
+    </head>
+
+    <body>
+        <div id="navbar">
+            <a href="#" id="about">About</a>
+            <a href="#" id="store">Store</a>
+            <a href="#" id="gmail">Gmail</a>
+            <a href="#" id="images">Images</a>
+            <a href="#" id="menu"
+                ><img src="menu-button.jpg" height="30px" width="28px"
+            /></a>
+            <a href="#" id="sign-in">Sign in</a>
+        </div>
+        <div id="main">
+            <a href="#" id="logo"
+                ><img
+                    src="https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_272x92dp.png"
+            /></a>
+            <input type="text" id="search" />
+            <div id="flexbox">
+                <button id="google-search-btn">Google Search</button
+                ><button
+                    id="feeling-lucky-btn"
+                    a
+                    href="#"
+                    onclick="randomSite();"
+                >
+                    I'm Feeling Lucky
+                </button>
+            </div>
+        </div>
+
+        <div id="footer">
+            <a href="#" id="advertising">Advertising</a>
+            <a href="#" id="business">Business</a>
+            <a href="#" id="how-search-works">How Search works</a>
+            <a href="#" id="privacy">Privacy</a>
+            <a href="#" id="terms">Terms</a>
+            <a href="#" id="settings">Settings</a>
+        </div>
+
+        <!-- script for the I'm feeling lucky button, when the button is clicked it will send to a random website from our search results -->
+        <script>
```

Comment:
good recovery to use hardcoded data, but if you need it would  be a better practice to keep the data in the separate file